base_report_to_printer/tests/test_printing_server.py

```diff
@@ -89,19 +89,6 @@ def test_update_printers_update_unavailable(self, cups):
         self.Model.action_update_printers()
         self.assertEqual("unavailable", rec_id.status)
 
-    @mock.patch("%s.cups" % model)
-    def test_update_archived_printers(self, cups):
-        """ It should update status even if printer is archived """
-        rec_id = self.new_printer()
-        rec_id.toggle_active()
-        self.server.refresh()
-        cups.Connection().getPrinters().get.return_value = False
-        self.Model.action_update_printers()
-        self.assertEqual(
-            "unavailable",
-            rec_id.status,
-        )
-
```

Comment:
Hummm this doesn't seem like a bad test that should be removed. Besides I see some changes in code related to this, so if you remove the test how do we know it is OK?

I mean: I don't understand why archived printers should still be searched, but if somebody took the time to even write a test for that, it seems like an important thing. 🤔 

Is there any reason to remove this?